dispatches/models/fossil_case/properties/hitecsalt_properties.py

```diff
@@ -0,0 +1,424 @@
+#        Cp in J/kg/K
+        cp_param = {1: 5806,
+                    2: -10.833,
+                    3: 7.2413E-3}
+        self.cp_param = Param(RangeSet(3), initialize=cp_param,
+                              doc="specific heat parameters")
+
+#        Density (rho) coefficients
+#        rho in kg/m3
+        rho_param = {1: 2293.6,
+                    2: -0.7497}
+        self.rho_param = Param(RangeSet(2), initialize=rho_param,
+                              doc="density parameters")
+        
+#        Dynamic Viscosity (mu) coefficients
+#        Mu in Pa.s
+        mu_param = {1: -4.343,
+                    2: -2.0143,
+                    3: -5.011}
+        self.mu_param = Param(RangeSet(3), initialize=mu_param,
+                              doc="dynamic viscosity parameters")
+        
+#        Thermal conductivity (kappa) coefficients
+#        kappa in W/(m.K)
+        kappa_param = {1: 0.421,
+                       2: -6.53E-4,
+                       3: -260}
+        self.kappa_param = Param(RangeSet(3), initialize=kappa_param,
+                                 doc="thermal conductivity parameters")
+
+#        Thermodynamic reference state
+#        ref_temperature in K
+        self.ref_temperature = Param(within=PositiveReals,
+                                     mutable=True,
+                                     default=298.15,
+                                     doc='Reference temperature [K]')
+
+    @classmethod
+    def define_metadata(cls,obj):
+        obj.add_properties({'flow_mass':{'method': None, 'units': 'kg/s'},
+                            'temperature':{'method': None, 'units': 'K'},
+                            'pressure':{'method': None, 'units': 'Pa'},
+                            'cp_specific_heat':{'method': None,
+                                                'units': 'J/kg/K'},
+                            'density':{'method': None, 'units': 'kg/m3'},
+                            'enthalpy':{'method': None, 'units': 'J/kg'},
+                            'dynamic_viscosity':{'method': None,
+                                                 'units': 'Pa.s'},
+                            'thermal_conductivity':{'method': None,
+                                                    'units': 'W/m/K'}})
+        obj.add_default_units({'time': pyunits.s,
+                               'length': pyunits.m,
+                               'mass': pyunits.kg,
+                               'amount': pyunits.mol,
+                               'temperature': pyunits.K})
+
+class _StateBlock(StateBlock):
+    """
+    This Class contains methods which should be applied to Property Blocks as a
+    whole, rather than individual elements of indexed Property Blocks.
+    """
+    def initialize(blk, state_args=None,
+                   outlvl=0, hold_state=False,
+                   state_vars_fixed=False,
+                   solver=None,
+                   optarg={}):
```

Comment:
`optarg=None` and `outlvl` should be updated to use the IDAES logger.

Reply:
addressed